brave-spark-framework-filter/src/test/webapp/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```

Comment:
hmm I don't think we want to rely on a webapp directory. is this something spark is creating? can it be changed to be a part of the "target" directory instead?